src/runtime/Icon.vue

```diff
@@ -59,7 +59,7 @@ watch(() => appConfig.nuxtIcon?.iconifyApiOptions, () => {
 const state = useState<Record<string, IconifyIcon | undefined>>('icons', () => ({}))
 const isFetching = ref(false)
 const iconName = computed(() => {
-  if (appConfig.nuxtIcon.aliases && appConfig.nuxtIcon.aliases[props.name]) {
+  if (appConfig.nuxtIcon?.aliases && appConfig.nuxtIcon.aliases[props.name]) {
```

Comment:
This can be further refactored with a single chain:
```suggestion
  if (appConfig.nuxtIcon?.aliases?.[props.name]) {
```